src/coreclr/hosts/unixcoreconsole/CMakeLists.txt

```diff
@@ -31,8 +31,4 @@ if(NOT CLR_CMAKE_PLATFORM_ANDROID)
   )
 endif()
 
-add_dependencies(coreconsole
-    coreclr
```

Comment:
@rahku 
>I don't this is true. Cross-component build should only be building dll/exe's listed in file https://github.com/dotnet/coreclr/blob/master/crosscomponents.cmake

Ah, right, I keep forgetting that we are still processing the CMakeLists.txt files but some targets are just not built since nothing needs them.